### ISSUE TITLE
Get rid of `content` folder in engine container image

### ISSUE THREAD

**Issue by buep** [Praqma/questionnaire-engine#169]
Today we have our engine image containing the `content` folder as the engine image also have deployment and is used for demonstration showing the demo-questionnaire.

This means when our content repositories uses the engine docker image, they will have to overwrite the content folder to avoid also serving the demo questionnaire content.
This is okay for now, but we want to avoid it in the future.

One way would be to create an engine container image that do not contain content, and then use it with a demo content repository generating a new demo container image that can be tested and finally used to verify the engine image is also working a release it.
So basically the engine image can't serve anything, unless you use it in a new container image with some content.

**Comment by edipetres** [comment]
Also add a fallback for the questionnaire-models Docker image since the default fallback (`template`) will be removed
